requirements/specifications/intents/user-interest.md

```diff
@@ -0,0 +1,290 @@
+that time **MUST** be ignored. The timeout value is stored in the
+device's configuration manifest.
+
+To be notified when a user expresses interest in the currently displayed
+content, an App **MUST** provide the
+`xrn:firebolt:capability:discovery:interest` capability by enabling the
+`Discovery.onRequestUserInterest` notification.
+
+If there is a valid entity to return, then the method registered by the
+App **MUST** return the currently displayed entity meta-data.
+
+If there is no valid entity to return, then the method **MUST** throw an
+exception.
+
+If the foreground App returns a valid `EntityInfo` before the timeout,
+then, the value from the **MUST** be used.
+
+Once a single `EntityInfo` is selected the platform **MUST** dispatch a
+`Content.onUserInterestedIn` notification to all Apps that have registered
+for it (typically Aggregated Experience Apps) with metadata about the
+app and entity as its data.
+
+The `Content.onUserInterestedIn` event has the `InterestedInIntent` as its
+data.
+
+The platform **MUST NOT** dispatch more than one `onUserInterestedIn`
+notification, even if more than one App responded to the pull request.
+
+An Aggregated Experience can register for the `Content.onUserInterestedIn`
+notification, and it will receive notifications when an `EntityInfo` is
+returned from the active App after a `Discovery.userInterest` call is
+fulfilled.
+
+## 5. User Interest Bulk Updates
+
+Sending bulk interest updates, e.g. Entities the user expressed interest
+in on a different platform, is not supported.
+
+## 6. Core SDK APIs
+
+The following APIs are exposed by the Firebolt Core SDK as part of the
+`Discovery` module.
+
+### 6.1. InterestType
+This is an enum with the following values:
+
+- `"interest"`
+- `"disinterest"`
+
+### 6.2. Discovery.userInterest
+
+This is a push API that allows Apps to push entities that the user has
+expressed interest in to the platform.
+
+To push an entity that the user is interested in pass an `EntityInfo`
+object to the method:
+
+```typescript
+Discovery.userInterest(type: InterestType, entity: EntityInfo): Promise<void>
+```
+
+### 6.3. Discovery Interest Provider
+To respond to requests for the current entity, because the user has
+expressed interest in some way that the platform manages, register a
+provider:
+
+```typescript
+interface IDiscoveryInterestProvider {
+  function userInterest(type: InterestType): Promise<EntityInfo>
+}
+
+Discovery.provide("xrn:firbolt:capability:discovery:interest", IDiscoveryInterestProvider)
+```
+
+### 6.4. InterestIntent
+
+An `InterestIntent` denotes that the user has expressed interest in the
+currently displayed and/or selected content:
+
+```typescript
+type InterestIntent {
+  action: "interest"
+  data: {
+    type: "interest"
+  },
+  context: {
+    source: "rcu" | "voice"
+  }
+}
+```
+
+## 7. Manage SDK APIs
+
+The following APIs are exposed by the Firebolt Core SDK as part of the
```

Comment:
typo: `Firebolt Core SDK` => `Firebolt Manage SDK`

Reply:
Actually, I guess it should be `Firebolt Discovery SDK` ?

Reply:
yeah, "Discovery," fixed.